contributor-docs/adrs/adr-xxx-behavior-isolation.md

```diff
@@ -0,0 +1,209 @@
+# ADR XXX: Isolating behaviors through custom elements and vanilla JavaScript
```

Comment:
I think we need a number here?

Reply:
Also, perhaps we should link to this ADR from 002, specifying that 002 was superseded?